cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/services/SourceControlManagementService.java

```diff
@@ -70,7 +72,7 @@
  * Service that manages source control for repositories and applications.
  * It exposes repository CRUD apis and source control tasks that do pull/pull/list applications in linked repository.
  */
-public class SourceControlManagementService {
+public class SourceControlManagementService extends AbstractIdleService {
```

Comment:
Do we need to make `SourceControlManagementService` an `AbstractIdleService`? We could make  `RepositoryCleanupService` an `AbstractIdleService` itself and start/stop in `AppFabricServer` , example: https://github.com/cdapio/cdap/pull/15007/files

Reply:
done